packages/common/src/intl/intlHelpers.ts

```diff
@@ -24,6 +26,81 @@ export const useTranslation = (ns?: Namespace): TypedTFunction<LocaleKey> => {
   return (key, options) => (key ? t(key, options) : '');
 };
 
+// This custom formatter will truncate cents to two digits when there are none.
+// However, still allows a large precision.
+// Without this custom formatter, a value of 12 would be formatted as
+// 12.00000 if the precision was 5, for example. With this change, the format
+// would be 12.00. With a precision of 2, then currency values would be
+// truncated to be a maximum of two decimal places.
+const formatter: currency.Format = (currency?, options?): string => {
+  const { decimal = '.' } = options ?? {};
+  if (!currency) return '';
+  if (!currency.cents()) {
+    return `${currency.value}${decimal}${'0'.repeat(2)}`;
+  }
+  return String(parseFloat(String(currency.value)));
+};
```

Comment:
I don't know if this is needed, but wanting to handle the case of tiny cent values, but also have normal looking money values

Reply:
Some tests would be helpful to see it in action

Reply:
oops - I wrote some and they were failing as this was silly. Not quite as simplistic as that 😁 

To get to where I wanted it would require a bit more work - specifically need to get the pattern from `options` and do a regex replace with the value, symbol and also handle the group separator and decimal 🙃